Dockerfile

```diff
@@ -49,6 +49,8 @@ ADD https://github.com/smarty-gettext/smarty-gettext/raw/master/function.locale.
 RUN chmod a+r /usr/share/php/smarty/libs/plugins/block.t.php /usr/share/php/smarty/libs/plugins/function.locale.php
 
 # Install site
-COPY . /var/www/html/
+COPY website/ /var/www/html/
 
-# Use it with: docker run -it --rm --name geokrety -p 80:80 -v $(pwd):/var/www/html/ geokrety
+RUN chmod a+rwx /var/www/html/templates/compile/ /var/www/html/templates/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer/
```

Comment:
Ho no! :( please do not use 0777 mode. Would be preferable to chown to `www-data`

Reply:
agree sure!

Reply:
I've added a commit :)